docker-compose-services/mysql/docker-compose.mysql.yaml

```diff
@@ -0,0 +1,28 @@
+version: "3.6"
+services:
+  mysql:
+    container_name: ddev-${DDEV_SITENAME}-mysql
+    labels:
+      com.ddev.site-name: ${DDEV_SITENAME}
+      com.ddev.approot: $DDEV_APPROOT
+    image: mysql:5.7
+    restart: "no"
+    environment:
+      MYSQL_ROOT_PASSWORD: "root"
+    ports:
+      # <Port exposed> : < MySQL Port running inside container>
+      - "3306"
+    volumes:
+      - type: "volume"
+        source: mysql-db 
+        target: "/var/lib/mysql"
+        volume:
+          nocopy: true
+      - type: "bind"
+        source: "."
+        target: "/mnt/ddev_config"
```

Comment:
Please add the line `    hostname: ${DDEV_SITENAME}-mysql`, which will give it a useful internal hostname, help people find their way around.